libraries/DataFlash/LogStructure.h

```diff
@@ -1039,6 +1041,7 @@ enum LogMessages {
     LOG_PIDY_MSG,
     LOG_PIDA_MSG,
     LOG_PIDS_MSG,
+    LOG_PIDL_MSG,
```

Comment:
Does the location of this, changing the enum order, matter? Inquiring minds want to know!